steps/install.sh

```diff
@@ -59,15 +59,15 @@ install_package xmlstarlet
 install_package gawk
 
 if "${LOCAL}/help/is-macos.sh"; then
-  brew install coreutils
-  brew install wget
+  sudo -u $SUDO_USER  brew install coreutils
+  sudo -u $SUDO_USER  brew install wget
 fi
 
 if ! pdftotext -v >/dev/null 2>&1; then
   if "${LOCAL}/help/is-linux.sh"; then
     "${LOCAL}/help/sudo.sh" apt-get install -y xpdf
   elif "${LOCAL}/help/is-macos.sh"; then
-    brew install poppler
+    sudo -u $SUDO_USER  brew install poppler
```

Comment:
@nai1ka maybe it's better to use `"${LOCAL}/help/sudo.sh"`, as we do in all other places?

Reply:
Sure, fixed this.
Now sudo.sh can be used with --as-user flag in order to run commands as a default user